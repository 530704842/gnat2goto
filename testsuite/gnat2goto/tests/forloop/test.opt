ALL SKIP CBMC seems to hang in a way that can't even be killed by --timeout
