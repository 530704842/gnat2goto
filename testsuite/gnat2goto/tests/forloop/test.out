CBMC version 5.6 64-bit x86_64 linux
Parsing forloop.json_symtab
Converting
Generating GOTO Program
Adding CPROVER library (x86_64)
Removal of function pointers and virtual functions
Partial Inlining
Generic Property Instrumentation
Starting Bounded Model Checking
Unwinding loop forloop.0 iteration 1 file forloop.adb line 5 column 20 thread 0
Unwinding loop forloop.0 iteration 2 file forloop.adb line 5 column 20 thread 0
Unwinding loop forloop.0 iteration 3 file forloop.adb line 5 column 20 thread 0
Unwinding loop forloop.1 iteration 1 file forloop.adb line 9 column 28 thread 0
Unwinding loop forloop.1 iteration 2 file forloop.adb line 9 column 28 thread 0
Unwinding loop forloop.1 iteration 3 file forloop.adb line 9 column 28 thread 0
Unwinding loop forloop.2 iteration 1 file forloop.adb line 13 column 23 thread 0
Unwinding loop forloop.2 iteration 2 file forloop.adb line 13 column 23 thread 0

simple slicing removed 0 assignments
Generated 0 VCC(s), 0 remaining after simplification
VERIFICATION SUCCESSFUL

